roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -225,6 +225,7 @@ kiali_defaults:
     port: 20001
     web_fqdn: ""
     web_history_mode: ""
+    web_port: 20001
```

Comment:
Is it true we want to hardcode the default to 20001? I read the description of this value and it says, "When empty, Kiali will try to guess this value from HTTP headers." As I understand, that's the current behavior (attempts to guess the port).

I'm just confirming that we no longer want "guessing" to be the default behavior and that we will now always assume 20001 by default unless the person sets this value to "" in the Kiali CR.

Reply:
I think we DO want the default behavior of "guessing" the public port.

I re-checked the changes in the back-end and I see that if "guessing" fails, it fallbacks to the listening port in the pod (which is 20001 by default).

So, I don't think we want a fixed value here. It will break existing installs that are relying on Kiali to "guess" the public port.